gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/computer/VertexProgram.java

```diff
@@ -68,7 +68,20 @@ public default void storeState(final Configuration configuration) {
      * @param graph         the graph that the VertexProgram will run against
      * @param configuration the configuration to load the state of the VertexProgram from.
      */
+    @Deprecated
```

Comment:
When we deprecate, the javadoc should also include something like:

```text
@deprecated As of release 3.2.8, replaced by {@link #loadState(Configuration, Graph)}
```

That syntax on the link isn't quite right because Graph is varargs, but hopefully you get the idea.

Reply:
Thanks, done.